conans/test/command/inspect_test.py

```diff
@@ -0,0 +1,101 @@
+        self.assertIn("There are no packages", client.out)
+
+    def name_version_test(self):
+        server = TestServer()
+        client = TestClient(servers={"default": server}, users={"default": [("lasote", "mypass")]})
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    name = "MyPkg"
+    version = "1.2.3"
+"""
+        client.save({"conanfile.py": conanfile})
+        client.run("inspect . -a=name")
+        self.assertIn("name: MyPkg", client.out)
+        client.run("inspect . -a=version")
+        self.assertIn("version: 1.2.3", client.out)
+        client.run("inspect . -a=version -a=name")
+        self.assertIn("name: MyPkg", client.out)
+        self.assertIn("version: 1.2.3", client.out)
+        client.run("inspect . -a=version -a=name --json=file.json")
+        contents = load(os.path.join(client.current_folder, "file.json"))
+        self.assertIn('"version": "1.2.3"', contents)
+        self.assertIn('"name": "MyPkg"', contents)
+
+        client.run("export . lasote/testing")
+        client.run("inspect MyPkg/1.2.3@lasote/testing -a=name")
+        self.assertIn("name: MyPkg", client.out)
+        client.run("inspect MyPkg/1.2.3@lasote/testing -a=version")
+        self.assertIn("version: 1.2.3", client.out)
+
+        client.run("upload MyPkg* --confirm")
+        client.run('remove "*" -f')
+        client.run("inspect MyPkg/1.2.3@lasote/testing -a=name -r=default")
+        self.assertIn("name: MyPkg", client.out)
+        client.run("inspect MyPkg/1.2.3@lasote/testing -a=version -r=default")
+        self.assertIn("version: 1.2.3", client.out)
+
+    def attributes_display_test(self):
+        client = TestClient()
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    settings = "os", "compiler", "arch"
+    pass
+"""
+        client.save({"conanfile.py": conanfile})
+        client.run("inspect . -a=short_paths")
+        self.assertIn("short_paths: False", client.out)
+        client.run("inspect . -a=name")
+        self.assertIn("name: None", client.out)
+        client.run("inspect . -a=version")
+        self.assertIn("version: None", client.out)
+        client.run("inspect . -a=settings")
+        self.assertIn("settings: ('os', 'compiler', 'arch')", client.out)
+
+        error = client.run("inspect . -a=unexisting_attr", ignore_error=True)
+        self.assertTrue(error)
+        self.assertIn("ERROR: 'Pkg' object has no attribute 'unexisting_attr'", client.out)
+
+    def options_test(self):
+        client = TestClient()
+        conanfile = """from conans import ConanFile
+class Pkg(ConanFile):
+    options = {"option1": [True, False], "option2": [1, 2, 3], "option3": "ANY"}
+    default_options = "option1=False", "option2=2", "option3=randomANY"
+"""
+        client.save({"conanfile.py": conanfile})
+        client.run("inspect . -a=options")
+        self.assertEquals(client.out, """options:
+option1: ['False', 'True'], default=False
```

Comment:
Shouldn't it be `option1: [False, True], default=False` ?

Reply:
The options values are always text values. I am not adding the quotes, it is the output because those values are actually strings. I am not sure how to improve that, because you cannot just unquote, because it is possible that the user already provided them as strings, and there is now way to know.